portal/server/sentry.server.config.ts

```diff
@@ -7,7 +7,7 @@
 
 import * as Sentry from "@sentry/nextjs";
 
-if (process.env.ENABLE_SENTRY === "production") {
+if (process.env.ENABLE_SENTRY === "yes") {
```

Comment:
Question: what is behind this change?

Reply:
Ignore this, the correct on now after the refactoring is `ConfigurationLoader.config.enableSentry = true | false`.